configuration/forms.md

```diff
@@ -6,12 +6,12 @@ There are two types of forms:
 - **JSON forms**: used for SMS interfaces such as formatted SMS, SIM applications, and Medic Collect*
 - **XForms**: used for forms used within the web app, whether it is accessed in browser or via the Android app.
 
-## JSON forms
+# JSON forms
 Used for SMS interfaces such as formatted SMS, SIM applications, and Medic Collect. You can view the list of JSON forms and load new ones through the webapp's Configuration pages, or via the `forms` field of `app_settings.json`. Each form has fields defined in our specific JSON format, eg:
 
 ```js
 {
-  "F": {
+  {
```

Comment:
Sorry, I've just noticed this. I'm not sure why you're removing the property name (`"F"`), the value is used when getting form configs (https://github.com/medic/medic/blob/master/api/src/controllers/record-utils.js#L183) and should match the `meta.code` value. Also, the json is now invalid. 
`meta.code` appears to only be used when parsing the message (https://github.com/medic/medic/blob/master/api/src/services/report/smsparser.js#L74).

Reply:
Ah, good catch. I saw that it was not used elsewhere, so thought it was a relic. Will revert.

Reply:
Since they have to be the same It would be nice to not have to have both!